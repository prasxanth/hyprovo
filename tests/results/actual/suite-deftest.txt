2021-11-24 17:25:52.104 - INFO - Pass ✓... (suite-deftest / test-add): add pass
2021-11-24 17:25:52.107 - ERROR - Fail ✗... (suite-deftest / test-add): add fail
2021-11-24 17:25:52.111 - INFO - Pass ✓... (suite-deftest / test-sub): sub pass
2021-11-24 17:25:52.113 - ERROR - Fail ✗... (suite-deftest / test-sub): sub fail
2021-11-24 17:25:52.113 - DEBUG - (suite-deftest): Test Summary → Total: 4    Pass: 2    Fail: 2
