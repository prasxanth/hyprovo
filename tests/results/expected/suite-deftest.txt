2021-11-24 17:24:55.681 - INFO - Pass ✓... (suite-deftest / test-add): add pass
2021-11-24 17:24:55.683 - ERROR - Fail ✗... (suite-deftest / test-add): add fail
2021-11-24 17:24:55.688 - INFO - Pass ✓... (suite-deftest / test-sub): sub pass
2021-11-24 17:24:55.691 - ERROR - Fail ✗... (suite-deftest / test-sub): sub fail
2021-11-24 17:24:55.691 - DEBUG - (suite-deftest): Test Summary → Total: 4    Pass: 2    Fail: 2
